product/roundhouse.console/Program.cs

```diff
@@ -298,6 +299,10 @@ private static void parse_arguments_and_set_up_configuration(ConfigurationProper
                 .Add("rcm=|recoverymode=",
                      "RecoveryMode - This instructs RH to set the database recovery mode to Simple|Full|NoChange. Defaults to NoChange.",
                      option => configuration.RecoveryMode = (RecoveryMode)Enum.Parse(typeof(RecoveryMode), option, true))
+                //user tokens
+                .Add("ut=|usertokens=",
+                    "UserTokens - This is a list of key/value pars used in scripts: the token '{{SomeToken}}' will be replace by 'value123' if 'ut=SomeToken=value123' is provide.",

```

Comment:
This doesn't mention they need to be passed with ampersand. I would prefer they are separated by something else, perhaps `;`?